pandas/tests/reductions/test_reductions.py

```diff
@@ -856,6 +856,16 @@ def test_idxmax(self):
         result = s.idxmin()
         assert result == 1.1
 
+        # GH#36566
+        s = Series("a", dtype="string").value_counts()
+        assert s.idxmax() == "a"
+
+        # GH#32749
+        from pandas.tests.extension.decimal import DecimalArray, make_data
+
+        s = Series(DecimalArray(make_data()[:1]))
```

Comment:
we need to generically test extension types, it is actually very tricky to do in this file. do in pandas/tests/extension/base/reduce.py

Reply:
Moved. I added a BaseArgReduceTests baseclass. This is the best way I can think of.